### PR TITLE
[DOCS] Remove license refs from install topics

### PR DESCRIPTION
**Pull request by debadair**

### CODE REVIEW

File: docs/reference/setup/install/deb.asciidoc [hunk 1]
```diff
@@ -105,36 +105,8 @@ endif::[]
 
 include::skip-set-kernel-parameters.asciidoc[]
 
-ifeval::["{release-state}"!="unreleased"]
-
-[NOTE]
 ==================================================
```
Comment (debadair) [inline docs/reference/setup/install/deb.asciidoc:108]:
```suggestion
```

File: docs/reference/setup/install/rpm.asciidoc [hunk 2]
```diff
@@ -96,36 +96,8 @@ must explicitly enable the repository as indicated in the sample commands above.
 
 endif::[]
 
-ifeval::["{release-state}"!="unreleased"]
-
-[NOTE]
 ==================================================
```
Comment (debadair) [inline docs/reference/setup/install/rpm.asciidoc:99]:
```suggestion
```

**Review by jrodewig — APPROVED** [review]
LGTM.

We also need to look at this section of `TESTING.asciidoc`:

https://github.com/elastic/elasticsearch/blob/master/TESTING.asciidoc#distribution

It's a pretty small section, but that may be worth a separate PR.

Thanks for handling this @debadair!

**Comment by mark-vieira** [comment]
@debadair I've already made most of these changes in my local branch related to the recent licensing changes. Let's get through the pipe first and then follow up on anything we might have missed as part of the review process for that PR so we aren't stepping on anyone's toes.

**Comment by mark-vieira** [comment]
> We also need to look at this section of TESTING.asciidoc:

I intend on addressing this as well.

**Comment by mark-vieira** [comment]
Thanks for the initiative on this Deb. I'm going to go ahead and close this for now though since the same changes will be pushed in my upcoming PR. I encourage you however to keep in mind any other changes that might need to be made here and reach out so that can get incorporated.

**Pull request closed without merging**